test/plots/stackedAreaPlotTests.ts

```diff
@@ -2,485 +2,346 @@
+        assert.strictEqual(stackedAreaPlot.y(accessor, linearScale), stackedAreaPlot, "setter returns calling object");
+        assert.strictEqual(stackedAreaPlot.y().accessor, accessor, "accessor set");
+        stackedAreaPlot.renderTo(svg);
+
+        let stackedAreas = stackedAreaPlot.content().selectAll("path");
+        assert.strictEqual(stackedAreas.size(),
+          stackedAreaPlot.datasets().length, "same number of area selections as datasets");
+
+        let stackedYs = calculateStackedYs(accessor);
+        stackedAreas.each(function(data, index) {
+          let stackedAreaSelection = d3.select(this);
+          let areaVertices = TestMethods.areaVertices(stackedAreaSelection);
+          let areaYs = areaVertices.map((areaVertex) => areaVertex.y).slice(0, -2);
+          data.forEach((datum: any, datumIndex: number) => {
+            let y = linearScale.scale(stackedYs[index][datumIndex]);
+            assert.closeTo(areaYs[datumIndex], y, window.Pixel_CloseTo_Requirement, "cools");
+          });
+        });
+        stackedAreaPlot.destroy();
+        svg.remove();
+      });
     });
 
-    it("pixel positions account for stack offsets", () => {
-      let dataYs = renderer.entities().map((entity) => yScale.invert(entity.position.y));
-      let dataset1Ys = dataset1.data().map((d) => d.y);
-      let dataset2Ys = dataset2.data().map((d, i) => d.y + dataset1.data()[i].y);
-      assert.includeMembers(dataYs, dataset1Ys, "all dataset1 points found");
-      assert.includeMembers(dataYs, dataset2Ys, "all dataset2 points found");
-      svg.remove();
+    describe("setting the y0 property", () => {
+      let stackedAreaPlot: Plottable.Plots.StackedArea<number>;
+
+      beforeEach(() => {
+        let data1 = [
+          {foo: 1},
+          {foo: 3}
+        ];
+        let dataset1 = new Plottable.Dataset(data1);
+        dataset1.metadata({ bar: 7 });
+        let data2 = [
+          {foo: 1},
+          {foo: 3}
+        ];
+        let dataset2 = new Plottable.Dataset(data2);
+        dataset2.metadata({ bar: 10 });
+        stackedAreaPlot = new Plottable.Plots.StackedArea<number>();
+        stackedAreaPlot.addDataset(dataset1);
+        stackedAreaPlot.addDataset(dataset2);
+        stackedAreaPlot.x((d, i) => i);
+        stackedAreaPlot.y((d, i) => i);
+      });
+
+      it("defaults to the 0 constant value", () => {
+        assert.strictEqual(stackedAreaPlot.y0().accessor(null, 0, null), 0, "property set to 0");
+        stackedAreaPlot.destroy();
+      });
+
+      it("can set the y0 property accessor to a constant value", () => {
+        let constantValue = 10;
+        stackedAreaPlot.y0(constantValue);
+        assert.strictEqual(stackedAreaPlot.y0().accessor(null, 0, null), constantValue, "property set to constant value");
+        stackedAreaPlot.destroy();
+      });
+
+      it("can set the y0 property accessor to be dependent on the input data", () => {
```

Comment:
I personally don't see too much immediate value in adding a skipped test.  I would much rather create an issue for the bug and then make the test at that point.